diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index dacd15e30d1710402dfc5cebf4ed0518e1fa64e4..29bfc4336ef3bbcfb6fa1f2a606138a210d950d8 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4012,6 +4012,7 @@ const FeatureEntry::FeatureVariation kWebAuthnAndroidCredManVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
