diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index 6d52b553b21b06a00c0853ce4f4cb02cf47fd7dd..412f5d44e364ae8686d33268e103a0629336833f 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2687,6 +2687,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
