diff --git a/chrome/browser/ui/browser_navigator.cc b/chrome/browser/ui/browser_navigator.cc
index ef1db072dde7379dd312e97d6cccc3bcfa655d1d..e0664506d59411c8ac039f523049ecce0bc4656b 100644
--- a/chrome/browser/ui/browser_navigator.cc
+++ b/chrome/browser/ui/browser_navigator.cc
@@ -586,6 +586,7 @@ base::WeakPtr<content::NavigationHandle> Navigate(NavigateParams* params) {
     // Block any navigation requests in locked fullscreen mode.
     return nullptr;
   }
+  BRAVE_ADJUST_NAVIGATE_PARAMS_FOR_URL
 
   // Open System Apps in their standalone window if necessary.
   // TODO(crbug.com/1096345): Remove this code after we integrate with intent
@@ -932,6 +933,7 @@ bool IsHostAllowedInIncognito(const GURL& url) {
   base::StringPiece host = url.host_piece();
   if (scheme != content::kChromeUIScheme)
     return true;
+  BRAVE_IS_HOST_ALLOWED_IN_INCOGNITO
 
   if (host == chrome::kChromeUIChromeSigninHost) {
 #if BUILDFLAG(IS_WIN)
