diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index f0aaa24344c71cac732e6fe27ed97fde507eaf54..f508f163ff74daa8a9810bc438f3b890d28d0948 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -3876,6 +3876,7 @@ const FeatureEntry::FeatureVariation kAndroidHubVariations[] = {
 //
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
