diff --git a/chrome/browser/resources/settings/router.ts b/chrome/browser/resources/settings/router.ts
index 84b2f6258a8709d529b34a3f36193405ecd43dd7..1d544df16d7a49fa5c34ebaf3f64173fe418bec9 100644
--- a/chrome/browser/resources/settings/router.ts
+++ b/chrome/browser/resources/settings/router.ts
@@ -104,6 +104,7 @@ export interface SettingsRoutes {
   SYSTEM: Route;
   TRACKING_PROTECTION: Route;
   TRIGGERED_RESET_DIALOG: Route;
+  GET_STARTED: Route; SHIELDS: Route; SOCIAL_BLOCKING: Route; REWARDS: Route; EXTENSIONS: Route; EXTENSIONS_V2: Route; BRAVE_SYNC: Route; BRAVE_SYNC_SETUP: Route; BRAVE_IPFS: Route; BRAVE_WALLET: Route; BRAVE_WEB3: Route; BRAVE_HELP_TIPS: Route; BRAVE_NEW_TAB: Route; THEMES: Route; BRAVE_SITE_SETTINGS_COOKIES_DETAILS: Route;
 }
 
 /** Class for navigable routes. */
