diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index a7463d5e82b73572d97e12f5335e2c54fd1c8ca8..2555607fbf1e89723c68d88afaed6ca46c8f0059 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -2694,6 +2694,7 @@ static_library("browser") {
       "metrics/family_link_user_metrics_provider.h",
     ]
   }
+  import("//brave/browser/sources.gni") sources += brave_chrome_browser_sources deps += brave_chrome_browser_deps public_deps += brave_chrome_browser_public_deps allow_circular_includes_from += brave_chrome_browser_allow_circular_includes_from
 
   if (!is_chromeos_ash) {
     sources += [
