diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index bf02e7b81d96c89c8da491c7e409acd3e47aa9e7..04be4720a9e76ef6ea7deabe81bacda94b5e0beb 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -297,6 +297,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
