diff --git a/ios/chrome/browser/sync/model/sync_service_factory.mm b/ios/chrome/browser/sync/model/sync_service_factory.mm
index d8141bd7754247813ff0e58f66a7d713a3f7b671..dff0ae5643c85886690768a8c7b1976201f0e965 100644
--- a/ios/chrome/browser/sync/model/sync_service_factory.mm
+++ b/ios/chrome/browser/sync/model/sync_service_factory.mm
@@ -97,7 +97,7 @@ std::unique_ptr<KeyedService> BuildSyncService(web::BrowserState* context) {
   init_params.debug_identifier = browser_state->GetDebugName();
 
   auto sync_service =
-      std::make_unique<syncer::SyncServiceImpl>(std::move(init_params));
+      BRAVE_BUILD_SERVICE_INSTANCE_FOR
   sync_service->Initialize();
 
   // TODO(crbug.com/1400663): Remove the workaround below once
