diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index 857270e552ca97c2e9e766c92714853f2c5bc709..1e4f04d8813f5d1c7e5061be0665d6ef58b6204c 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -357,6 +357,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterIntegerPref(kAppStoreRatingTotalDaysOnChromeKey, 0);
   registry->RegisterListPref(kAppStoreRatingActiveDaysInPastWeekKey);
   registry->RegisterTimePref(kAppStoreRatingLastShownPromoDayKey, base::Time());
+  BRAVE_REGISTER_LOCAL_STATE_PREFS
 }
 
 void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -565,6 +566,7 @@ void RegisterBrowserStatePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterDoublePref(kTimeSpentInFeedAggregateKey, 0.0);
 
   registry->RegisterBooleanPref(kSyncRequested, false);
+  BRAVE_REGISTER_BROWSER_STATE_PREFS
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -745,6 +747,7 @@ void MigrateObsoleteBrowserStatePrefs(PrefService* prefs) {
       prefs->SetDouble(kTimeSpentInFeedAggregateKey, value);
     }
   }
+  BRAVE_MIGRATE_OBSOLETE_BROWSER_STATE_PREFS
 }
 
 void MigrateObsoleteUserDefault(void) {
