diff --git a/net/cookies/canonical_cookie.cc b/net/cookies/canonical_cookie.cc
index dbaeff68b212bf5652846f4e4ba17bf066ea9d47..471f50fce7a8e05b85ba4dc5fb258ca2686e428d 100644
--- a/net/cookies/canonical_cookie.cc
+++ b/net/cookies/canonical_cookie.cc
@@ -515,6 +515,7 @@ base::Time CanonicalCookie::ValidateAndAdjustExpiryDate(
     // * network_handler.cc::MakeCookieFromProtocolValues
     fixed_creation_date = base::Time::Now();
   }
+  BRAVE_CANONICAL_COOKIE_VALIDATE_AND_ADJUST_EXPIRY_DATE
   base::Time maximum_expiry_date = fixed_creation_date + base::Days(400);
   if (expiry_date > maximum_expiry_date) {
     return maximum_expiry_date;
